source/_integrations/smlight.markdown

```diff
@@ -40,14 +40,17 @@ You need a supported SLZB-06 adapter.
 
 The following sensors will be created:
 
-- **Core Temperature** - Temperature of core ESP32
-- **Zigbee Temperature** - Temperature of Zigbee CC2652 or EFR32 chip
-- **Core Uptime** - Uptime of Core device
-- **Zigbee Uptime** - Uptime of Zigbee connection to ZHA/Z2M
-- **RAM Usage** - Monitor RAM Usage
-- **FS Usage** - Monitor filesystem usage
+- **Core temperature** - Temperature of core ESP32
+- **Zigbee temperature** - Temperature of Zigbee CC2652 or EFR32 chip
+- **Core uptime** - Uptime of Core device
+- **Zigbee uptime** - Uptime of Zigbee connection to ZHA/Z2M
+- **RAM usage** - Monitor RAM Usage
+- **FS usage** - Monitor filesystem usage
+- **Connection mode** -  Connection mode - Ethernet, Wi-Fi, or USB
```

Comment:
what does this connection mode do?

Reply:
This how the SLZB-06x device is connected. USB mode it acts like a USB Zigbee dongle, but the web interface (and  HA connection) falls back to ethernet or wifi if available.